pkg/flowkit/config/config_test.go

```diff
@@ -145,55 +145,58 @@ func Test_GetContractsForNetworkComplex(t *testing.T) {
 	market, err := conf.Contracts.ByName("KittyItemsMarket")
 	assert.NoError(t, err)
 
-	assert.Equal(t, kitty.Name, "KittyItems")
-	assert.Equal(t, market.Source, "./cadence/kittyItemsMarket/contracts/KittyItemsMarket.cdc")
+	assert.Equal(t, "KittyItems", kitty.Name)
+	assert.Equal(t, "./cadence/kittyItemsMarket/contracts/KittyItemsMarket.cdc", market.Source)
 }
 
 func Test_GetContractsByNameAndNetworkComplex(t *testing.T) {
 	conf := generateComplexConfig()
 	market, err := conf.Contracts.ByNameAndNetwork("KittyItemsMarket", "testnet")
 	assert.NoError(t, err)
 
-	assert.Equal(t, market.Source, "0x123123123")
+	assert.Equal(t, "0x123123123", market.Source)
 }
 
 func Test_GetContractsByNetworkComplex(t *testing.T) {
 	conf := generateComplexConfig()
 	contracts := conf.Contracts.ByNetwork("emulator")
 
-	assert.Equal(t, 5, len(contracts))
-	assert.Equal(t, contracts[0].Name, "NonFungibleToken")
-	assert.Equal(t, contracts[1].Name, "FungibleToken")
-	assert.Equal(t, contracts[2].Name, "Kibble")
-	assert.Equal(t, contracts[3].Name, "KittyItems")
-	assert.Equal(t, contracts[4].Name, "KittyItemsMarket")
+	assert.Len(t, contracts, 5)
```

Comment:
ohh I had this on my mind so long, thank you!!! coming from JS world this order always confuses me

Reply:
Yeah, when we first adopted testify in Cadence, I was confused by the argument order, but got used to it. Good thing is now failures are not confusing anymore. 

The `require/assert.Len` seems odd though, because it's inconsistent with `Equal`'s expected/actual order.